src/WebJobs.Script/Host/ScriptHostManager.cs

```diff
@@ -132,21 +154,33 @@ private static void LogErrors(ScriptHost host)
             }
         }
 
-        // Let the existing host instance finish currently executing functions.
-        private async Task Orphan(ScriptHost instance)
+        /// <summary>
```

Comment:
@mathewc can you take a look at this comment doc and let me know if you're happy with the verbiage there?